### ISSUE TITLE
Use additional global view based on combined view

### ISSUE THREAD

**Issue by liuyuchen6666** [orashi/Fast-MoCo#3]
Thanks for your great work. It is an very interesting perspective for SSL. I just have a question that have you ever tried to add the additional global view to generate "p"? I mean besides the 6 local views, also generate a global view as conventionl, which leads to 7 total views. Would the performance be improved? I am very curious about this. I would appreciate it if you would share your opinion. Thanks a lot for your help!

**Comment by orashi** [comment]
Thanks for your interest! 

I believe what you are referring to can be implemented in 2 ways.
1) a global view by forwarding an additional 224x224 image.
This is similar to "Fast-MoCo w/ Multi-Crop" setting we described in Section 5.3, Table 4(b)(iii), and Section B.5. It utilizes 2x6 local views and 1 global view, improving the 100ep. ImageNet Linear Evaluation performance by 0.7 (73.5->74.2). Nevertheless, this requires an additional forward&backward path for a 224x224 image which increases the per iteration training cost by roughly 40%.
2) a global view by combining all patches
This is not mentioned in the paper, but we actually did this experiment before and it seems the additional all-patches-included sample wasn't of any help.
